Model/Sales/TrackTraceHolder.php

```diff
@@ -254,19 +272,73 @@ private function getValueOfOption($options, $optionKey)
 
     /**
      * @param $shipmentId
-     * @return \Magento\Sales\Model\ResourceModel\Order\Collection
+     *
+     * @return array
      */
     private function getItemsCollectionByShipmentId($shipmentId)
     {
+        /** @var \Magento\Framework\App\ResourceConnection $connection */
         $connection = $this->objectManager->create('\Magento\Framework\App\ResourceConnection');
         $conn = $connection->getConnection();
         $select = $conn->select()
             ->from(
-                ['main_table' => 'sales_shipment_item']
+                ['main_table' => $connection->getTableName('sales_shipment_item')]
             )
             ->where('main_table.parent_id=?', $shipmentId);
         $items = $conn->fetchAll($select);
 
         return $items;
     }
+
+    /**
+     * @param Order\Shipment\Track $magentoTrack
+     * @param int $totalWeight
+     *
+     * @return TrackTraceHolder
+     * @throws LocalizedException
+     * @throws \Exception
+     */
+    private function calculateTotalWeight($magentoTrack, $totalWeight = 0)
+    {
+        if ($this->consignment->getPackageType() !== AbstractConsignment::PACKAGE_TYPE_DIGITAL_STAMP) {
+            return $this;
+        }
+
+        if ($totalWeight > 0) {
+            $this->consignment->setPhysicalProperties(["weight" => $totalWeight]);
+
+            return $this;
+        }
+
+        $weightFromSettings = (int)self::$defaultOptions->getDigitalStampWeight();
```

Comment:
```suggestion
        $weightFromSettings = (int) self::$defaultOptions->getDigitalStampWeight();
```
je moet je phpstorm maar even goed instellen

Reply:
Done